tests/foreman/sys/test_hooks.py

```diff
@@ -0,0 +1,159 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Foreman Hooks
+
+:Requirement: Foreman Hooks
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: SYS
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+"""
+from fauxfactory import gen_ipaddr
+from nailgun import entities
+from requests.exceptions import HTTPError
+from robottelo import ssh
+from robottelo.decorators import run_in_one_thread, tier3
+from robottelo.test import TestCase
+from robottelo.datafactory import valid_hosts_list, valid_hostgroups_list
+
+HOOKS_DIR = '/usr/share/foreman/config/hooks'
+LOGS_DIR = '/usr/share/foreman/tmp/hooks.log'
+
+
+@run_in_one_thread
+class ForemanHooksTestCase(TestCase):
+    """Test class for testing foreman hooks"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create logger script to be executed via hooks"""
+        super(ForemanHooksTestCase, cls).setUpClass()
+        cls.org = 1  # using Default_Organization
```

Comment:
Any specific reason to run this test against the default organisation?

Reply:
I did't want to create a separate org for this for performance reasons, hooks should react to actions accross orgs, I thought I could use what we already have for these basic trigger actions.